stargaze/assetlist.json

```diff
@@ -566,6 +566,56 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stargaze/images/vault.png"
         }
       ]
+    },
+    {
+      "description": "Liquid token STARS from NFT lending platform GopLend",
```

Comment:
Sorry, I'm not quite understanding how this is derived. This is a liquid form of what position, exactly?

Reply:
Thanks for your attention, yes, this is a liquid form of lended STARS, i.e. like stSTARS from Stride (just not for staking)

This token is issued in exchange for the $STARS token when a user makes a deposit on the platform

Reply:
Thanks for explaining. I was wondering if this aught to describe how it derives from STARS using traces[], but there is currently no trace type to describe liquid-lending (I presume this works kinda like bonds).